website/source/docs/providers/aws/r/emr_cluster.html.md

```diff
@@ -67,6 +67,7 @@ The following arguments are supported:
 * `release_label` - (Required) The release label for the Amazon EMR release
 * `master_instance_type` - (Required) The EC2 instance type of the master node
 * `service_role` - (Required) IAM role that will be assumed by the Amazon EMR service to access AWS resources
+* `security_configuration` - (Optional) The security configuration name to attach to the EMR cluster, the security configuration cannot be created through Terraform. Only valid for EMR clusters with `release_label` 4.8.0 or greater
```

Comment:
derp, forgot to remove that. That's from the old PR I took over, which didn't include the resource, and was created before I added the resource last Friday :D